_posts/2016/2016-04-12-hr-quality-assurance.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title: "Working as a QA Engineer at ePages"
+date: "2016-04-12 07:16:17"
+icon: book
+categories: working-at-epages
+authors: ["Birgit, Serena", "Vlad"]
+---
+
+In one of our latest posts, you already learned about what it’s like to work as a [Ruby Developer](https://developer.epages.com/blog/2016/02/04/hr-ruby.html) and a [Java Developer](https://developer.epages.com/blog/2016/03/01/hr-java.html) at ePages.
+To give you some insight into our world of Quality Assurance, we've talked to one of our Quality Assurance Engineers to get some information about his daily work.
+
+## Here's the story:
+
+{% image blog/blog-qa-1.jpg 55% right %}{% endimage %}
+
+Hi, my name is Vlad.
+I'm 28 years old and I come from Kharkiv, Ukraine.
+I'm working as a Quality Assurance Engineer at ePages since September 2015.
+
+### How did you end up working at ePages?
+
+Due to the poor conditions in the labour market, I was looking for a job outside of Ukraine.
```

Comment:
I would not say it's because of the poor conditions in the labour market. I think, Ukraine has even more open IT positions.
The main problem is the "war" in Ukraine and poor living conditions.